### DISCUSSION TITLE
Resultado do Front-End Tooling Survey 2018

### DISCUSSION THREAD

**Discussion by didiraja** [frontendbr/forum#1079] [Discussão]
Saíram os resultados do [Front-End Tooling Survey 2018](https://ashleynolan.co.uk/blog/frontend-tooling-survey-2018-results). Vou postar aqui pra gente fomentar a discussão.

Esses resultados não são regra de nada, mas tem alguns pontos interessantes sobre a realidade da comunidade.

- A experiência geral da comunidade passeia entre 2-5 e 5-10 anos com número de 10-15 sendo maior que os 0-2 (essa aqui foi uma surpresa, eu podia jurar que o majoritário seria entre 0-5 anos)
- Sass está dominando com folga absurda o mundo dos pré-processadores
- Bootstrap é largamente usado, mas a quantidade de devs que não usam nenhum framework é quase a mesma
- jQuery ainda é o framework JS que os devs mais se sentem confortáveis de usar
- Mas independente desse conforto, React segue pouco atrás do jQuery em uso
- Webpack domina com folga entre os bundlers

**Comment by tarciozemel** [comment]
Lembrando que o resultado foi com base nas respostas de 5.097 pessoas.

**Comment by didiraja** [comment]
@tarciozemel sim, mas pelo que falaram foi a pesquisa com mais respostas desde 2016

**Comment by felipefialho** [comment]
Alguns comentários:

- Stylus com apenas ~2% de uso, atrás de (apenas) PostCSS e até de LESS 😱 
- ~51% não usa nenhuma metodologias CSS... doidera 😕 
-  CSS-in-JS é cada vez mais realidade 🔥 
- NPM Scripts como o _task runner_ mais usado me surpreende, gosto 🥇 
- jQuery perdeu 20% de uso, reflexo da "migração" para SPAs e JavaScript puro 🤙 
- Webpack tomou de assalto o mundo de _module bundlers_ 🔪 
- Galera ta usando _transpilers_ sem medo de ser feliz 😄

**Comment by klarkc** [comment]
Interessante ver o crescimento do Vue de 2016 pra 2017
